index.js

```diff
@@ -90,15 +96,23 @@ async function fetchImageUrl(url) {
   try {
     const response = await axios(url);
     const $ = cheerio.load(response.data);
-    const singleImage = $('.page-mainimage').find('img').attr('src');
-    const doubleImage = $(
-      '.swiper-articleimage li.swiper-slide[data-hash="slide1"]',
-    )
-      .find('img.swiper-lazy')
-      .attr('src');
-    const imageUrl = singleImage || doubleImage;
-
-    return `${BASE_URL}${imageUrl}`;
+    const singleImageElement = $('.page-mainimage').find('img');
+    const multipleImageElement = $('.swiper-wrapper').find('img');
```

Comment:
I've found an issue here. They might have multiple carousels in a page. I'm using this one as reference: https://www.berlin.de/en/news/6158006-5559700-kadewe-karstadt-may-reopen-their-entire-.en.html
This select returns a bunch of images instead of two. I would recommend you to first select the `main content` (article) from the page and then select the images. You can also use `.swiper-articleimage` as it will select just that carousel.